m3/BUILD

```diff
@@ -0,0 +1,24 @@
+cc_library(
+    name = "m3",
```

Comment:
nit: Can we move the m3 reporter into `reporter/m3` or rename it `m3reporter` or something?  It seems like a vague name and its not clear where consequent reporters would go in the project.

Reply:
sgtm, will udpate

Reply:
On second thought, I think it might be preferable to add a `Contributing` section to the README explaining the reporters so we can keep each of the libraries in separate top-level directories.